docs/DECLARING_DEPENDENCIES.md

```diff
@@ -0,0 +1,204 @@
+- [Cloud Secret Manager](https://github.com/googleapis/java-secretmanager)
+- [Cloud Security Center](https://github.com/googleapis/java-securitycenter)
+- [Cloud Security Scanner](https://github.com/googleapis/java-websecurityscanner)
+- [Cloud Talent Solution](https://github.com/googleapis/java-talent)
+- [Cloud Web Risk](https://github.com/googleapis/java-webrisk)
+- [Dialogflow](https://github.com/googleapis/java-dialogflow)
+- [Kubernetes Engine](https://github.com/googleapis/java-container)
+- [reCAPTCHA Enterprise](https://github.com/googleapis/java-recaptchaenterprise)
+- [Stackdriver Error Reporting](https://github.com/googleapis/java-errorreporting)
+- [Cloud Compute](https://github.com/googleapis/java-compute)
+- [Cloud DNS](https://github.com/googleapis/java-dns)
+- [Cloud Logging via Logback](https://github.com/googleapis/java-logging-logback)
+- [Cloud Resource Manager](https://github.com/googleapis/java-resourcemanager)
+- [Cloud Storage via NIO](https://github.com/googleapis/java-storage-nio)
+
+`com.google.cloud:libraries-bom` also covers several other libraries
+that client code does not usually depend on directly. However if you do import
+any of these, use  `com.google.cloud:libraries-bom` to specify their versions
+too:
+
+- [GAX Google API Extensions for Java](https://github.com/googleapis/gax-java)
+- [Google Auth Library](https://github.com/googleapis/google-auth-library-java)
+- [Google API Client Library for Java](https://github.com/googleapis/google-api-java-client)
+- [Grafeas Artifact Metadata API](https://github.com/grafeas/grafeas)
+
+## Ensuring Compatibility
+
+To ensure that your own project uses compatible versions of these
+libraries, follow the guidance below for your build system.
+
+### Maven
+
+#### Use the requireUpperBoundDeps enforcer rule
+
+Maven's dependency mediation algorithm can select older versions
+instead of newer versions, meaning new features will be missing that
+other libraries depend on. The `requireUpperBoundDeps` enforcer rule
+can be used to automatically discover incorrect version selection
+because it fails the build when an older version is chosen instead of
+a newer one.
+
+You can add `requireUpperBoundDeps` to your build like this:
+
+```
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M3</version>
+        <executions>
+          <execution>
+            <id>enforce</id>
+            <goals>
+              <goal>enforce</goal>
+            </goals>
+            <configuration>
+              <rules>
+                <requireUpperBoundDeps/>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+```
+
+When the newest versions are not compatible, you can use BOMs to
+select compatible versions.
+
+#### Import BOMs to ensure dependencies are consistent
```

Comment:
```suggestion
#### Import GCP Libraries BOM to ensure dependencies are consistent
```

This section "Ensuring Compatibility" is written for generic usage of BOMs (not specific to libraries-bom), while this document is "Declaring Dependencies on Google Cloud Java Libraries". Is this intentional?

Reply:
done

Reply:
done